storage/file_test.go

```diff
@@ -1,6 +1,7 @@
 package storage
 
 import (
+	"azure-sdk-for-go/storage"
```

Comment:
Everything in Travis. Look at the lines before the one in red (in this case 344 - 350). That is the failed command output.